packages/react/src/components/CalendarRange/CalendarRange.tsx

```diff
@@ -27,29 +60,38 @@ export const CalendarRange = createComponent<'div', CalendarRangeProps>((props,
     locale,
     visibleDuration: { months: 1 },
     createCalendar,
-  });
+  } as RangeCalendarStateOptions);
 
   const { calendarProps, nextButtonProps, prevButtonProps } = useRangeCalendar(
-    other,
+    other as RangeCalendarProps<DateValue>,
     state,
     calendarRef,
   );
 
-  const { className } = useStyles({ css });
+  const { className } = useStyles({ css, showCalendar });
+  const definedRanges: DefinedRange[] = showRanges && ranges ? ranges : getDefaultRanges();
```

Comment:
I think this should just be a default prop value in `CalendarRanges` component.

Reply:
sure

Reply:
sure